### PR TITLE
Unfinished business from #1084

### PR DESCRIPTION
**Pull request by rosskarchner**
- Fixes a few wording problems pointed out by @anselmbradford and @sebworks.
- removes Sheer from Travis build
- Fixes an issue with macropolo-- we're now using our own "Environment" class, not one of the ones provided in macropolo.environments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,15 +102,15 @@ To do this, run the following:
 # Use the cfgov-refresh virtualenv.
 workon cfgov-refresh
```
Comment (anselmbradford) [inline README.md]: Should `workon` come after `cd`'ing into the project directory or does it not matter?

**Comment by anselmbradford** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (rosskarchner) [inline README.md]: It doesn't matter

**Comment by kave** [comment]
:+1:

File: .travis.yml [hunk 2]
```diff
@@ -25,7 +19,8 @@ install:
   - npm install -g bower
 
   # Project initialization.
-  - sheer index
+  - pip install -r requirements/test.txt 
+  - python cfgov/manage.py sheer_index
```
Comment (kurtw) [inline .travis.yml]: Should we throw in something about `--reindex`?

Comment (rosskarchner) [inline .travis.yml]: I don't think Travis care's-- but maybe in the README ;)

**Pull request merged**